data/quick-builds/rebel-alliance.json

```diff
@@ -58,7 +58,7 @@
       "threat": 3,
       "pilots": [
         {
-          "id": "wegeantilles",
+          "id": "wedgeantilles",
```

Comment:
We should be able to detect these automatically. Perhaps we can write a test that checks if all ids in the Quick Builds exist in the JSON files.